integration/sandbox_clean_remove_test.go

```diff
@@ -124,3 +124,108 @@ func TestSandboxRemoveWithoutIPLeakage(t *testing.T) {
 	t.Logf("Should not be able to find the pod ip in host-local checkpoint")
 	assert.False(t, checkIP(ip))
 }
+
+// Issue: https://github.com/containerd/containerd/issues/5768
+// PR: https://github.com/containerd/containerd/pull/5904
+// This test simulates the network setup failure and teardown failure by renaming the name of portmap binary.
+// Due to the nature of chained cni plugin, host-local ipam should have allocated IP address.
+// RunPodSandx should leave such sandbox in the metadata store to cleanup IP address later.
+func TestNetworkTeardownFailureWithoutIPLeakage(t *testing.T) {
```

Comment:
As discussed offline with Fuwei, I will delete this test and use his integration test with failpoint mechanism.